cli/integration-test/entrypoint.clj

```diff
@@ -75,15 +73,15 @@
 
   (apply require namespaces)
 
-  (let [timeout-minutes 10
+  (let [timeout-minutes 16
```

Comment:
> Increases integration-tests timeout to 16min, req response timeout to 90sec and log request/response duration.

This bothers me a little bit, why do we really need that?

Reply:
The windows reqs take 2.5 times longer to complete compared to Ubuntu. The major culprit is the initialization request which involves calling the build project tool to collect the classpaths. 

I haven't looked it into the detail (clj-async-profile does not work on windows), but I believe the slowness on windows is a combination of slow command invocation (I know this is a general problem with windows), slower file access times, perhaps a lower spec CI machine as well?

I have produced some data collected from the successful ci run earlier (simply craws through the integration-test log files and extracts the request/response timings). I've removed ubuntu/11/15/17 since their timins are similar to that of ubuntu/8. Let me know if you need me to investigate further.


```
** logs_12120/jvm-integration-test (windows-latest, 8)/7_Run integration tests.txt
| :id |   :sec | :req                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
|-----+--------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|   1 | 32.343 | 2022-09-16T06:53:14.8905330Z 2022-09-16T06:53:14.8820365 Client 1 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   1 |  0.143 | 2022-09-16T06:53:48.0080561Z 2022-09-16T06:53:47.8459773 Client 1 sending request: {:jsonrpc "2.0", :method :shutdown, :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
|   2 |  0.223 | 2022-09-16T06:53:48.0782646Z 2022-09-16T06:53:48.0652764 Client 2 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   2 |  0.005 | 2022-09-16T06:54:01.2553554Z 2022-09-16T06:54:00.6801902 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/a.clj"}, :position {:line 9, :character 9}}, :id 3}                                                                                                                                                                                                                                                                                       |
|   2 |  0.004 | 2022-09-16T06:54:01.2556801Z 2022-09-16T06:54:00.6801902 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 3, :character 4}}, :id 4}                                                                                                                                                                                                                                                                                       |
|   2 |  0.026 | 2022-09-16T06:54:01.2560192Z 2022-09-16T06:54:00.6801902 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 5, :character 2}}, :id 5}                                                                                                                                                                                                                                                                                       |
|   2 |  0.149 | 2022-09-16T06:54:01.2562954Z 2022-09-16T06:54:00.8051917 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 7, :character 2}}, :id 6}                                                                                                                                                                                                                                                                                       |
|   2 |  0.017 | 2022-09-16T06:54:01.2565826Z 2022-09-16T06:54:00.9615242 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 9, :character 2}}, :id 7}                                                                                                                                                                                                                                                                                       |
|   2 |  0.035 | 2022-09-16T06:54:01.2569191Z 2022-09-16T06:54:00.9771487 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 11, :character 2}}, :id 8}                                                                                                                                                                                                                                                                                      |
|   3 | 12.613 | 2022-09-16T06:54:01.3072741Z 2022-09-16T06:54:01.3052735 Client 3 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   3 |  0.178 | 2022-09-16T06:54:14.2121283Z 2022-09-16T06:54:13.9849791 Client 3 sending request: {:jsonrpc "2.0", :method :textDocument/declaration, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/declaration/b.clj"}, :position {:line 4, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                     |
|   3 |  0.026 | 2022-09-16T06:54:14.2130466Z 2022-09-16T06:54:14.1734356 Client 3 sending request: {:jsonrpc "2.0", :method :textDocument/declaration, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/declaration/b.clj"}, :position {:line 6, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                     |
|   4 | 12.569 | 2022-09-16T06:54:14.2598053Z 2022-09-16T06:54:14.2515637 Client 4 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   4 |  0.027 | 2022-09-16T06:54:27.2701155Z 2022-09-16T06:54:26.9992649 Client 4 sending request: {:jsonrpc "2.0", :method :textDocument/implementation, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/implementation/a.clj"}, :position {:line 3, :character 3}}, :id 2}                                                                                                                                                                                                                                                                               |
|   5 | 12.547 | 2022-09-16T06:54:27.3120506Z 2022-09-16T06:54:27.2968774 Client 5 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   5 |  0.546 | 2022-09-16T06:54:41.1702784Z 2022-09-16T06:54:40.5689474 Client 5 sending request: {:jsonrpc "2.0", :method :textDocument/hover, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/text_change/a.clj"}, :position {:line 4, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                           |
|   5 |  0.011 | 2022-09-16T06:54:42.1691377Z 2022-09-16T06:54:41.64885 Client 5 sending request: {:jsonrpc "2.0", :method :textDocument/hover, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/text_change/a.clj"}, :position {:line 4, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                             |
|   6 | 12.981 | 2022-09-16T06:54:42.1698994Z 2022-09-16T06:54:41.7113539 Client 6 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}, :capabilities {:window {:showDocument {:support true}}, :workspace {:workspaceEdit {:documentChanges true}}}}, :id 1}                                                                                                                                                                                     |
|   6 |  0.411 | 2022-09-16T06:54:55.4866695Z 2022-09-16T06:54:54.6992645 Client 6 sending request: {:jsonrpc "2.0", :method :textDocument/codeAction, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/code_action/a.clj"}, :context {:diagnostics []}, :range {:start {:line 5, :character 4}}}, :id 2}                                                                                                                                                                                                                                                    |
|   6 |  0.026 | 2022-09-16T06:54:55.6983492Z 2022-09-16T06:54:55.6368825 Client 6 sending request: {:jsonrpc "2.0", :method :workspace/executeCommand, :params {:command "drag-forward", :arguments ("file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/code_action/a.clj" 5 4)}, :id 3}                                                                                                                                                                                                                                                                                                   |
|   7 | 12.747 | 2022-09-16T06:54:56.2984597Z 2022-09-16T06:54:56.2934005 Client 7 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}, :capabilities {:textDocument {:completion {:contextSupport true, :completionItem {:snippetSupport true, :commitCharactersSupport true, :preselectSupport true, :documentationFormat ["markdown" "plaintext"], :resolveSupport {:properties ["documentation" "detail" "additionalTextEdits"]}}}}}}, :id 1} |
|   7 |  0.232 | 2022-09-16T06:55:09.7912383Z 2022-09-16T06:55:09.3783606 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 2, :character 3}}, :id 2}                                                                                                                                                                                                                                                                                       |
|   7 |  0.137 | 2022-09-16T06:55:09.9143444Z 2022-09-16T06:55:09.6127378 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 2, :character 4}}, :id 3}                                                                                                                                                                                                                                                                                       |
|   7 |  0.010 | 2022-09-16T06:55:09.9162804Z 2022-09-16T06:55:09.753365 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 4, :character 3}}, :id 4}                                                                                                                                                                                                                                                                                        |
|   8 | 12.789 | 2022-09-16T06:55:10.0441796Z 2022-09-16T06:55:10.0346279 Client 8 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   9 | 12.425 | 2022-09-16T06:55:23.4994625Z 2022-09-16T06:55:23.4963285 Client 9 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  10 | 12.441 | 2022-09-16T06:55:36.6493415Z 2022-09-16T06:55:36.6392394 Client 10 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :linters {:clj-kondo {:report-duplicates false}}}}, :id 1}                                                                                                                                                                                                                                                |
|  11 | 12.408 | 2022-09-16T06:55:51.1371535Z 2022-09-16T06:55:50.0960404 Client 11 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  12 | 12.504 | 2022-09-16T06:56:03.5754309Z 2022-09-16T06:56:03.5655102 Client 12 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :clj-depend {:layers {:a {:defined-by ".*\\.depend\\.a", :accessed-by-layers #{}}, :b {:defined-by ".*\\.depend\\.b", :accessed-by-layers #{:c}}}}}}, :id 1}                                                                                                                                              |
|  13 | 12.580 | 2022-09-16T06:56:16.8194367Z 2022-09-16T06:56:16.8157238 Client 13 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  14 | 12.584 | 2022-09-16T06:56:31.6504671Z 2022-09-16T06:56:31.6459724 Client 14 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  14 |  0.693 | 2022-09-16T06:56:45.0869580Z 2022-09-16T06:56:44.2348108 Client 14 sending request: {:jsonrpc "2.0", :method :textDocument/formatting, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/formatting.clj"}, :options {:tabSize 2, :insertSpaces true}}, :id 2}                                                                                                                                                                                                                                                                                |
|  14 |  0.075 | 2022-09-16T06:56:45.0874562Z 2022-09-16T06:56:44.9849381 Client 14 sending request: {:jsonrpc "2.0", :method :textDocument/rangeFormatting, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/formatting.clj"}, :options {:tabSize 2, :insertSpaces true}, :range {:start {:line 6, :character 4}, :end {:line 7, :character 11}}}, :id 3}                                                                                                                                                                                                   |
|  15 | 12.486 | 2022-09-16T06:56:45.1903704Z 2022-09-16T06:56:45.1880688 Client 15 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  15 |  0.041 | 2022-09-16T06:56:58.3611490Z 2022-09-16T06:56:58.0375492 Client 15 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 15, :character 0}, :newName "::click-now"}, :id 2}                                                                                                                                                                                                                                                                    |
|  16 | 12.578 | 2022-09-16T06:56:58.4219038Z 2022-09-16T06:56:58.4130595 Client 16 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  16 |  0.201 | 2022-09-16T06:57:11.4678434Z 2022-09-16T06:57:11.2460323 Client 16 sending request: {:jsonrpc "2.0", :method :textDocument/prepareRename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 6}}, :id 2}                                                                                                                                                                                                                                                                                     |
|  16 |  0.008 | 2022-09-16T06:57:11.4690068Z 2022-09-16T06:57:11.4335363 Client 16 sending request: {:jsonrpc "2.0", :method :textDocument/prepareRename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 15}}, :id 3}                                                                                                                                                                                                                                                                                    |
|  17 | 12.386 | 2022-09-16T06:57:11.5126424Z 2022-09-16T06:57:11.4960782 Client 17 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  17 |  0.161 | 2022-09-16T06:57:24.5754800Z 2022-09-16T06:57:24.1422963 Client 17 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 1, :character 36}, :newName "spec"}, :id 2}                                                                                                                                                                                                                                                                           |
|  18 | 12.774 | 2022-09-16T06:57:24.7374629Z 2022-09-16T06:57:24.721023 Client 18 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  18 |  0.357 | 2022-09-16T06:57:38.1571432Z 2022-09-16T06:57:37.7448478 Client 18 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 15}, :newName ":your-key"}, :id 2}                                                                                                                                                                                                                                                                     |
|  19 | 12.976 | 2022-09-16T06:57:38.3140235Z 2022-09-16T06:57:38.30795 Client 19 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                    |
|  19 |  0.817 | 2022-09-16T06:57:53.3340540Z 2022-09-16T06:57:51.495709 Client 19 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/b.cljc"}, :position {:line 3, :character 4}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                        |
|  20 | 12.733 | 2022-09-16T06:57:53.3792680Z 2022-09-16T06:57:53.3729783 Client 20 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  20 |  0.074 | 2022-09-16T06:58:07.7395439Z 2022-09-16T06:58:06.4860845 Client 20 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 3, :character 9}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                       |
|  21 | 13.875 | 2022-09-16T06:58:12.2804608Z 2022-09-16T06:58:07.7738782 Client 21 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  21 |  0.446 | 2022-09-16T06:58:22.4922884Z 2022-09-16T06:58:21.826646 Client 21 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/single_a.clj"}, :position {:line 2, :character 0}, :newName "::click-now"}, :id 2}                                                                                                                                                                                                                                                                |
|  22 | 12.992 | 2022-09-16T06:58:22.5067156Z 2022-09-16T06:58:22.4984726 Client 22 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  22 |  0.367 | 2022-09-16T06:58:36.3622022Z 2022-09-16T06:58:35.8127004 Client 22 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/b.cljc"}, :position {:line 1, :character 52}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                      |
|  23 | 13.357 | 2022-09-16T06:58:36.4126624Z 2022-09-16T06:58:36.4069099 Client 23 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  23 |  0.153 | 2022-09-16T06:58:50.2273057Z 2022-09-16T06:58:49.7574147 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 7, :character 2}}, :id 2}                                                                                                                                                                                                                                                                       |
|  23 |  0.006 | 2022-09-16T06:58:50.2277700Z 2022-09-16T06:58:49.929292 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 2, :character 25}}, :id 3}                                                                                                                                                                                                                                                                       |
|  23 |  0.009 | 2022-09-16T06:58:50.2280405Z 2022-09-16T06:58:49.9449198 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 14, :character 4}}, :id 4}                                                                                                                                                                                                                                                                      |
|  24 | 13.921 | 2022-09-16T06:58:50.2881028Z 2022-09-16T06:58:50.2732992 Client 24 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  24 |  0.015 | 2022-09-16T06:59:05.8695045Z 2022-09-16T06:59:04.4150024 Client 24 sending request: {:jsonrpc "2.0", :method :textDocument/documentSymbol, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_symbol/a.clj"}}, :id 2}                                                                                                                                                                                                                                                                                                                |
|  25 | 13.802 | 2022-09-16T06:59:05.9310104Z 2022-09-16T06:59:05.9168753 Client 25 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  25 |  0.010 | 2022-09-16T06:59:20.5799425Z 2022-09-16T06:59:20.0747107 Client 25 sending request: {:jsonrpc "2.0", :method :textDocument/linkedEditingRange, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/linked_editing_range/a.cljc"}, :position {:line 2, :character 13}}, :id 2}                                                                                                                                                                                                                                                                  |
|  25 |  0.015 | 2022-09-16T06:59:20.5807809Z 2022-09-16T06:59:20.0921325 Client 25 sending request: {:jsonrpc "2.0", :method :textDocument/linkedEditingRange, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/linked_editing_range/a.cljc"}, :position {:line 2, :character 7}}, :id 3}                                                                                                                                                                                                                                                                   |
|  26 | 15.392 | 2022-09-16T06:59:20.6462100Z 2022-09-16T06:59:20.6373688 Client 26 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  26 |  0.164 | 2022-09-16T06:59:36.4882584Z 2022-09-16T06:59:36.0351713 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 1, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                     |
|  26 |  0.014 | 2022-09-16T06:59:36.4886435Z 2022-09-16T06:59:36.1914192 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 3, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                     |
|  26 |  0.007 | 2022-09-16T06:59:36.4892347Z 2022-09-16T06:59:36.2070573 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 2, :character 8}}, :id 4}                                                                                                                                                                                                                                                                                     |
|  27 | 14.682 | 2022-09-16T06:59:36.6426025Z 2022-09-16T06:59:36.6305695 Client 27 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false}}, :id 1}                                                                                                                                                                                                                                                                                                               |
|  27 |  0.697 | 2022-09-16T06:59:52.6162105Z 2022-09-16T06:59:51.3185624 Client 27 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 7, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  28 | 14.990 | 2022-09-16T06:59:52.8271336Z 2022-09-16T06:59:52.8195655 Client 28 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :dependency-scheme "jar", :java {:decompile-jar-as-project? false}}}, :id 1}                                                                                                                                                                                                                                           |
|  28 |  2.567 | 2022-09-16T07:00:09.8712720Z 2022-09-16T07:00:07.8082106 Client 28 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 8, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  28 |  0.007 | 2022-09-16T07:00:12.3868104Z 2022-09-16T07:00:10.3795017 Client 28 sending request: {:jsonrpc "2.0", :method "clojure/dependencyContents", :params {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/.lsp/.cache/java/decompiled/clojure/lang/PersistentVector.java"}, :id 3}                                                                                                                                                                                                                                                                                                       |
|  29 | 12.998 | 2022-09-16T07:00:12.5041242Z 2022-09-16T07:00:12.4939376 Client 29 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :dependency-scheme "zipfile", :java {:decompile-jar-as-project? false}}}, :id 1}                                                                                                                                                                                                                                       |
|  29 |  2.758 | 2022-09-16T07:00:29.1090102Z 2022-09-16T07:00:25.4967883 Client 29 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 8, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  29 |  0.015 | 2022-09-16T07:00:29.1098093Z 2022-09-16T07:00:28.2568819 Client 29 sending request: {:jsonrpc "2.0", :method "clojure/dependencyContents", :params {:uri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/.lsp/.cache/java/decompiled/clojure/lang/PersistentVector.java"}, :id 3}                                                                                                                                                                                                                                                                                                       |
|  30 | 18.964 | 2022-09-16T07:00:30.1657882Z 2022-09-16T07:00:29.1789524 Client 30 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :stubs {:generation {:namespaces #{"datomic.api"}}}}}, :id 1}                                                                                                                                                                                                                                             |
|  31 | 19.284 | 2022-09-16T07:00:53.8129514Z 2022-09-16T07:00:53.8075179 Client 31 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  31 |  0.279 | 2022-09-16T07:01:13.3835719Z 2022-09-16T07:01:13.091759 Client 31 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|  32 | 36.761 | 2022-09-16T07:01:13.4464799Z 2022-09-16T07:01:13.4349099 Client 32 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-boot/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                             |
|  32 |  0.152 | 2022-09-16T07:01:50.3807010Z 2022-09-16T07:01:50.2012848 Client 32 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
|  33 | 58.597 | 2022-09-16T07:01:50.4359691Z 2022-09-16T07:01:50.4205035 Client 33 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-shadow/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                           |
|  33 |  0.043 | 2022-09-16T07:02:49.0815585Z 2022-09-16T07:02:49.0202448 Client 33 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
|  34 | 22.191 | 2022-09-16T07:02:49.1980511Z 2022-09-16T07:02:49.1927911 Client 34 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-lein/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                             |
|  34 |  0.183 | 2022-09-16T07:03:12.5968640Z 2022-09-16T07:03:11.3865918 Client 34 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
|  35 | 31.646 | 2022-09-16T07:03:16.9692859Z 2022-09-16T07:03:12.6834939 Client 35 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///D:/a/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-bb/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                               |
|  35 |  0.250 | 2022-09-16T07:03:45.4312139Z 2022-09-16T07:03:44.3365238 Client 35 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
** logs_12120/jvm-integration-test (macos-latest, 8)/7_Run integration tests.txt
| :id |   :sec | :req                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|-----+--------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|   1 | 13.700 | 2022-09-16T06:51:23.2653550Z 2022-09-16T06:51:23.264318 Client 1 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   1 |  0.004 | 2022-09-16T06:51:37.5159460Z 2022-09-16T06:51:37.511692 Client 1 sending request: {:jsonrpc "2.0", :method :shutdown, :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
|   2 |  6.181 | 2022-09-16T06:51:37.6755110Z 2022-09-16T06:51:37.675068 Client 2 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   2 |  0.069 | 2022-09-16T06:51:43.9070280Z 2022-09-16T06:51:43.870115 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/a.clj"}, :position {:line 11, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                      |
|   2 |  0.005 | 2022-09-16T06:51:43.9674700Z 2022-09-16T06:51:43.939159 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/a.clj"}, :position {:line 9, :character 9}}, :id 3}                                                                                                                                                                                                                                                                                       |
|   2 |  0.005 | 2022-09-16T06:51:43.9880100Z 2022-09-16T06:51:43.944879 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 3, :character 4}}, :id 4}                                                                                                                                                                                                                                                                                       |
|   2 |  0.016 | 2022-09-16T06:51:44.0019320Z 2022-09-16T06:51:43.950606 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 5, :character 2}}, :id 5}                                                                                                                                                                                                                                                                                       |
|   2 |  0.014 | 2022-09-16T06:51:44.0021770Z 2022-09-16T06:51:43.966712 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 7, :character 2}}, :id 6}                                                                                                                                                                                                                                                                                       |
|   2 |  0.009 | 2022-09-16T06:51:44.0024010Z 2022-09-16T06:51:43.981781 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 9, :character 2}}, :id 7}                                                                                                                                                                                                                                                                                       |
|   2 |  0.009 | 2022-09-16T06:51:44.0026310Z 2022-09-16T06:51:43.991021 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 11, :character 2}}, :id 8}                                                                                                                                                                                                                                                                                      |
|   3 |  6.400 | 2022-09-16T06:51:44.0983100Z 2022-09-16T06:51:44.097913 Client 3 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   3 |  0.064 | 2022-09-16T06:51:50.5529570Z 2022-09-16T06:51:50.503167 Client 3 sending request: {:jsonrpc "2.0", :method :textDocument/declaration, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/declaration/b.clj"}, :position {:line 4, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                     |
|   3 |  0.008 | 2022-09-16T06:51:50.5779270Z 2022-09-16T06:51:50.567521 Client 3 sending request: {:jsonrpc "2.0", :method :textDocument/declaration, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/declaration/b.clj"}, :position {:line 6, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                     |
|   4 |  6.426 | 2022-09-16T06:51:50.6095140Z 2022-09-16T06:51:50.609151 Client 4 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   4 |  0.063 | 2022-09-16T06:51:57.0759230Z 2022-09-16T06:51:57.038255 Client 4 sending request: {:jsonrpc "2.0", :method :textDocument/implementation, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/implementation/a.clj"}, :position {:line 3, :character 3}}, :id 2}                                                                                                                                                                                                                                                                               |
|   5 |  6.457 | 2022-09-16T06:51:57.2263950Z 2022-09-16T06:51:57.225652 Client 5 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   5 |  0.018 | 2022-09-16T06:52:04.2315540Z 2022-09-16T06:52:04.231143 Client 5 sending request: {:jsonrpc "2.0", :method :textDocument/hover, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/text_change/a.clj"}, :position {:line 4, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                           |
|   5 |  0.009 | 2022-09-16T06:52:05.3749390Z 2022-09-16T06:52:05.374492 Client 5 sending request: {:jsonrpc "2.0", :method :textDocument/hover, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/text_change/a.clj"}, :position {:line 4, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                           |
|   6 |  6.485 | 2022-09-16T06:52:05.4171600Z 2022-09-16T06:52:05.416622 Client 6 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}, :capabilities {:window {:showDocument {:support true}}, :workspace {:workspaceEdit {:documentChanges true}}}}, :id 1}                                                                                                                                                                                     |
|   6 |  0.088 | 2022-09-16T06:52:11.9216260Z 2022-09-16T06:52:11.903258 Client 6 sending request: {:jsonrpc "2.0", :method :textDocument/codeAction, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/code_action/a.clj"}, :context {:diagnostics []}, :range {:start {:line 5, :character 4}}}, :id 2}                                                                                                                                                                                                                                                    |
|   6 |  0.005 | 2022-09-16T06:52:12.1173100Z 2022-09-16T06:52:12.115871 Client 6 sending request: {:jsonrpc "2.0", :method :workspace/executeCommand, :params {:command "drag-forward", :arguments ("file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/code_action/a.clj" 5 4)}, :id 3}                                                                                                                                                                                                                                                                                                   |
|   7 |  6.381 | 2022-09-16T06:52:12.7309090Z 2022-09-16T06:52:12.730235 Client 7 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}, :capabilities {:textDocument {:completion {:contextSupport true, :completionItem {:snippetSupport true, :commitCharactersSupport true, :preselectSupport true, :documentationFormat ["markdown" "plaintext"], :resolveSupport {:properties ["documentation" "detail" "additionalTextEdits"]}}}}}}, :id 1} |
|   7 |  0.097 | 2022-09-16T06:52:19.1606890Z 2022-09-16T06:52:19.1188 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 2, :character 3}}, :id 2}                                                                                                                                                                                                                                                                                         |
|   7 |  0.013 | 2022-09-16T06:52:19.2334930Z 2022-09-16T06:52:19.217006 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 2, :character 4}}, :id 3}                                                                                                                                                                                                                                                                                       |
|   7 |  0.019 | 2022-09-16T06:52:19.2436930Z 2022-09-16T06:52:19.231023 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 4, :character 3}}, :id 4}                                                                                                                                                                                                                                                                                       |
|   8 |  6.516 | 2022-09-16T06:52:19.3195610Z 2022-09-16T06:52:19.319167 Client 8 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   9 |  6.489 | 2022-09-16T06:52:26.4576550Z 2022-09-16T06:52:26.457208 Client 9 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  10 |  6.422 | 2022-09-16T06:52:33.7153110Z 2022-09-16T06:52:33.714837 Client 10 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :linters {:clj-kondo {:report-duplicates false}}}}, :id 1}                                                                                                                                                                                                                                                |
|  11 |  6.341 | 2022-09-16T06:52:40.8413750Z 2022-09-16T06:52:40.840931 Client 11 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  12 |  6.459 | 2022-09-16T06:52:47.7849440Z 2022-09-16T06:52:47.77944 Client 12 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :clj-depend {:layers {:a {:defined-by ".*\\.depend\\.a", :accessed-by-layers #{}}, :b {:defined-by ".*\\.depend\\.b", :accessed-by-layers #{:c}}}}}}, :id 1}                                                                                                                                               |
|  13 |  6.484 | 2022-09-16T06:52:54.9165570Z 2022-09-16T06:52:54.916214 Client 13 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  14 |  6.425 | 2022-09-16T06:53:03.8255910Z 2022-09-16T06:53:03.825236 Client 14 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  14 |  0.085 | 2022-09-16T06:53:10.2947930Z 2022-09-16T06:53:10.254169 Client 14 sending request: {:jsonrpc "2.0", :method :textDocument/formatting, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/formatting.clj"}, :options {:tabSize 2, :insertSpaces true}}, :id 2}                                                                                                                                                                                                                                                                                |
|  14 |  0.022 | 2022-09-16T06:53:10.3501480Z 2022-09-16T06:53:10.3437 Client 14 sending request: {:jsonrpc "2.0", :method :textDocument/rangeFormatting, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/formatting.clj"}, :options {:tabSize 2, :insertSpaces true}, :range {:start {:line 6, :character 4}, :end {:line 7, :character 11}}}, :id 3}                                                                                                                                                                                                     |
|  15 |  6.333 | 2022-09-16T06:53:10.4731200Z 2022-09-16T06:53:10.472732 Client 15 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  15 |  0.170 | 2022-09-16T06:53:16.8491560Z 2022-09-16T06:53:16.810797 Client 15 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 15, :character 0}, :newName "::click-now"}, :id 2}                                                                                                                                                                                                                                                                    |
|  16 |  6.421 | 2022-09-16T06:53:17.0198460Z 2022-09-16T06:53:17.019445 Client 16 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  16 |  0.144 | 2022-09-16T06:53:23.4760660Z 2022-09-16T06:53:23.447181 Client 16 sending request: {:jsonrpc "2.0", :method :textDocument/prepareRename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 6}}, :id 2}                                                                                                                                                                                                                                                                                     |
|  16 |  0.006 | 2022-09-16T06:53:23.5972780Z 2022-09-16T06:53:23.59117 Client 16 sending request: {:jsonrpc "2.0", :method :textDocument/prepareRename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 15}}, :id 3}                                                                                                                                                                                                                                                                                     |
|  17 |  6.458 | 2022-09-16T06:53:23.7083700Z 2022-09-16T06:53:23.70786 Client 17 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  17 |  0.124 | 2022-09-16T06:53:30.2313100Z 2022-09-16T06:53:30.176213 Client 17 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 1, :character 36}, :newName "spec"}, :id 2}                                                                                                                                                                                                                                                                           |
|  18 |  6.396 | 2022-09-16T06:53:30.4038980Z 2022-09-16T06:53:30.403492 Client 18 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  18 |  0.158 | 2022-09-16T06:53:36.8740390Z 2022-09-16T06:53:36.807707 Client 18 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 15}, :newName ":your-key"}, :id 2}                                                                                                                                                                                                                                                                     |
|  19 |  6.462 | 2022-09-16T06:53:36.9885420Z 2022-09-16T06:53:36.988112 Client 19 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  19 |  0.194 | 2022-09-16T06:53:43.4923050Z 2022-09-16T06:53:43.454616 Client 19 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/b.cljc"}, :position {:line 3, :character 4}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                       |
|  20 |  6.484 | 2022-09-16T06:53:43.7093580Z 2022-09-16T06:53:43.708974 Client 20 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  20 |  0.152 | 2022-09-16T06:53:50.2379630Z 2022-09-16T06:53:50.199236 Client 20 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 3, :character 9}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                       |
|  21 |  6.448 | 2022-09-16T06:53:50.4572650Z 2022-09-16T06:53:50.456825 Client 21 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  21 |  0.171 | 2022-09-16T06:53:56.9596250Z 2022-09-16T06:53:56.910776 Client 21 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/single_a.clj"}, :position {:line 2, :character 0}, :newName "::click-now"}, :id 2}                                                                                                                                                                                                                                                               |
|  22 |  6.553 | 2022-09-16T06:53:57.1779670Z 2022-09-16T06:53:57.177543 Client 22 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  22 |  0.154 | 2022-09-16T06:54:03.7593840Z 2022-09-16T06:54:03.739338 Client 22 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/b.cljc"}, :position {:line 1, :character 52}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                      |
|  23 |  6.461 | 2022-09-16T06:54:03.9902730Z 2022-09-16T06:54:03.98985 Client 23 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  23 |  0.060 | 2022-09-16T06:54:10.4847440Z 2022-09-16T06:54:10.452176 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 7, :character 2}}, :id 2}                                                                                                                                                                                                                                                                       |
|  23 |  0.005 | 2022-09-16T06:54:10.5273530Z 2022-09-16T06:54:10.523275 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 2, :character 25}}, :id 3}                                                                                                                                                                                                                                                                      |
|  23 |  0.005 | 2022-09-16T06:54:10.5476540Z 2022-09-16T06:54:10.534499 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 14, :character 4}}, :id 4}                                                                                                                                                                                                                                                                      |
|  24 |  6.331 | 2022-09-16T06:54:10.6406620Z 2022-09-16T06:54:10.640279 Client 24 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  24 |  0.077 | 2022-09-16T06:54:17.0210530Z 2022-09-16T06:54:16.976937 Client 24 sending request: {:jsonrpc "2.0", :method :textDocument/documentSymbol, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_symbol/a.clj"}}, :id 2}                                                                                                                                                                                                                                                                                                                |
|  25 |  6.458 | 2022-09-16T06:54:17.2005120Z 2022-09-16T06:54:17.200067 Client 25 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  25 |  0.085 | 2022-09-16T06:54:23.6953050Z 2022-09-16T06:54:23.665935 Client 25 sending request: {:jsonrpc "2.0", :method :textDocument/linkedEditingRange, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/linked_editing_range/a.cljc"}, :position {:line 2, :character 13}}, :id 2}                                                                                                                                                                                                                                                                  |
|  25 |  0.007 | 2022-09-16T06:54:23.7597220Z 2022-09-16T06:54:23.751204 Client 25 sending request: {:jsonrpc "2.0", :method :textDocument/linkedEditingRange, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/linked_editing_range/a.cljc"}, :position {:line 2, :character 7}}, :id 3}                                                                                                                                                                                                                                                                   |
|  26 |  6.424 | 2022-09-16T06:54:23.8012210Z 2022-09-16T06:54:23.800847 Client 26 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  26 |  0.045 | 2022-09-16T06:54:30.2459980Z 2022-09-16T06:54:30.226905 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 1, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                     |
|  26 |  0.020 | 2022-09-16T06:54:30.2931360Z 2022-09-16T06:54:30.272662 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 3, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                     |
|  26 |  0.015 | 2022-09-16T06:54:30.3123900Z 2022-09-16T06:54:30.293666 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 2, :character 8}}, :id 4}                                                                                                                                                                                                                                                                                     |
|  27 |  6.341 | 2022-09-16T06:54:30.3502850Z 2022-09-16T06:54:30.349868 Client 27 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false}}, :id 1}                                                                                                                                                                                                                                                                                                               |
|  27 |  0.068 | 2022-09-16T06:54:36.7234370Z 2022-09-16T06:54:36.691505 Client 27 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 7, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  28 |  6.464 | 2022-09-16T06:54:36.7919010Z 2022-09-16T06:54:36.791469 Client 28 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :dependency-scheme "jar", :java {:decompile-jar-as-project? false}}}, :id 1}                                                                                                                                                                                                                                           |
|  28 |  1.321 | 2022-09-16T06:54:43.2659990Z 2022-09-16T06:54:43.256591 Client 28 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 8, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  28 |  0.010 | 2022-09-16T06:54:44.5881090Z 2022-09-16T06:54:44.577904 Client 28 sending request: {:jsonrpc "2.0", :method "clojure/dependencyContents", :params {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/.lsp/.cache/java/decompiled/clojure/lang/PersistentVector.java"}, :id 3}                                                                                                                                                                                                                                                                                                       |
|  29 |  6.488 | 2022-09-16T06:54:44.6597340Z 2022-09-16T06:54:44.659333 Client 29 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :dependency-scheme "zipfile", :java {:decompile-jar-as-project? false}}}, :id 1}                                                                                                                                                                                                                                       |
|  29 |  1.457 | 2022-09-16T06:54:51.2035040Z 2022-09-16T06:54:51.152633 Client 29 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 8, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  29 |  0.007 | 2022-09-16T06:54:52.6202170Z 2022-09-16T06:54:52.610072 Client 29 sending request: {:jsonrpc "2.0", :method "clojure/dependencyContents", :params {:uri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/.lsp/.cache/java/decompiled/clojure/lang/PersistentVector.java"}, :id 3}                                                                                                                                                                                                                                                                                                       |
|  30 | 10.135 | 2022-09-16T06:54:52.7162860Z 2022-09-16T06:54:52.715836 Client 30 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :stubs {:generation {:namespaces #{"datomic.api"}}}}}, :id 1}                                                                                                                                                                                                                                             |
|  31 |  9.741 | 2022-09-16T06:55:08.4770310Z 2022-09-16T06:55:08.476712 Client 31 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  31 |  0.009 | 2022-09-16T06:55:18.2288730Z 2022-09-16T06:55:18.21845 Client 31 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|  32 | 41.782 | 2022-09-16T06:55:18.2521170Z 2022-09-16T06:55:18.251733 Client 32 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-boot/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                             |
|  32 |  0.007 | 2022-09-16T06:56:00.0442550Z 2022-09-16T06:56:00.03404 Client 32 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|  33 | 32.272 | 2022-09-16T06:56:00.0831020Z 2022-09-16T06:56:00.082678 Client 33 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-shadow/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                           |
|  33 |  0.010 | 2022-09-16T06:56:32.3596490Z 2022-09-16T06:56:32.35802 Client 33 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|  34 | 12.127 | 2022-09-16T06:56:32.4166180Z 2022-09-16T06:56:32.416137 Client 34 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-lein/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                             |
|  34 |  0.009 | 2022-09-16T06:56:44.5499570Z 2022-09-16T06:56:44.54326 Client 34 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|  35 | 18.929 | 2022-09-16T06:56:44.5987830Z 2022-09-16T06:56:44.598269 Client 35 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///Users/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-bb/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                               |
|  35 |  0.033 | 2022-09-16T06:57:03.5461010Z 2022-09-16T06:57:03.529637 Client 35 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
** logs_12120/jvm-integration-test (ubuntu-latest, 8)/7_Run integration tests.txt
| :id |   :sec | :req                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
|-----+--------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|   1 | 11.764 | 2022-09-16T06:50:08.4867592Z 2022-09-16T06:50:08.484809 Client 1 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   1 |  0.005 | 2022-09-16T06:50:20.7820929Z 2022-09-16T06:50:20.781785 Client 1 sending request: {:jsonrpc "2.0", :method :shutdown, :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
|   2 |  5.361 | 2022-09-16T06:50:20.8034835Z 2022-09-16T06:50:20.802183 Client 2 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   2 |  0.072 | 2022-09-16T06:50:26.1843310Z 2022-09-16T06:50:26.169628 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/a.clj"}, :position {:line 11, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                      |
|   2 |  0.007 | 2022-09-16T06:50:26.2438728Z 2022-09-16T06:50:26.241906 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/a.clj"}, :position {:line 9, :character 9}}, :id 3}                                                                                                                                                                                                                                                                                       |
|   2 |  0.004 | 2022-09-16T06:50:26.2500866Z 2022-09-16T06:50:26.249684 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 3, :character 4}}, :id 4}                                                                                                                                                                                                                                                                                       |
|   2 |  0.015 | 2022-09-16T06:50:26.2544445Z 2022-09-16T06:50:26.254068 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 5, :character 2}}, :id 5}                                                                                                                                                                                                                                                                                       |
|   2 |  0.021 | 2022-09-16T06:50:26.2715629Z 2022-09-16T06:50:26.269898 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 7, :character 2}}, :id 6}                                                                                                                                                                                                                                                                                       |
|   2 |  0.010 | 2022-09-16T06:50:26.2930074Z 2022-09-16T06:50:26.291487 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 9, :character 2}}, :id 7}                                                                                                                                                                                                                                                                                       |
|   2 |  0.013 | 2022-09-16T06:50:26.3030945Z 2022-09-16T06:50:26.301505 Client 2 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/definition/b.clj"}, :position {:line 11, :character 2}}, :id 8}                                                                                                                                                                                                                                                                                      |
|   3 |  5.326 | 2022-09-16T06:50:26.3393335Z 2022-09-16T06:50:26.338024 Client 3 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   3 |  0.076 | 2022-09-16T06:50:31.6698575Z 2022-09-16T06:50:31.669196 Client 3 sending request: {:jsonrpc "2.0", :method :textDocument/declaration, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/declaration/b.clj"}, :position {:line 4, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                     |
|   3 |  0.007 | 2022-09-16T06:50:31.7469065Z 2022-09-16T06:50:31.745279 Client 3 sending request: {:jsonrpc "2.0", :method :textDocument/declaration, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/declaration/b.clj"}, :position {:line 6, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                     |
|   4 |  5.365 | 2022-09-16T06:50:31.7678245Z 2022-09-16T06:50:31.767481 Client 4 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   4 |  0.098 | 2022-09-16T06:50:37.1376152Z 2022-09-16T06:50:37.135592 Client 4 sending request: {:jsonrpc "2.0", :method :textDocument/implementation, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/implementation/a.clj"}, :position {:line 3, :character 3}}, :id 2}                                                                                                                                                                                                                                                                               |
|   5 |  5.261 | 2022-09-16T06:50:37.3005805Z 2022-09-16T06:50:37.300372 Client 5 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   5 |  0.016 | 2022-09-16T06:50:43.0633402Z 2022-09-16T06:50:43.062626 Client 5 sending request: {:jsonrpc "2.0", :method :textDocument/hover, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/text_change/a.clj"}, :position {:line 4, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                           |
|   5 |  0.006 | 2022-09-16T06:50:43.5810423Z 2022-09-16T06:50:43.580671 Client 5 sending request: {:jsonrpc "2.0", :method :textDocument/hover, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/text_change/a.clj"}, :position {:line 4, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                           |
|   6 |  5.057 | 2022-09-16T06:50:43.6638058Z 2022-09-16T06:50:43.663554 Client 6 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}, :capabilities {:window {:showDocument {:support true}}, :workspace {:workspaceEdit {:documentChanges true}}}}, :id 1}                                                                                                                                                                                     |
|   6 |  0.158 | 2022-09-16T06:50:48.7325736Z 2022-09-16T06:50:48.721712 Client 6 sending request: {:jsonrpc "2.0", :method :textDocument/codeAction, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/code_action/a.clj"}, :context {:diagnostics []}, :range {:start {:line 5, :character 4}}}, :id 2}                                                                                                                                                                                                                                                    |
|   6 |  0.005 | 2022-09-16T06:50:49.0760714Z 2022-09-16T06:50:49.073055 Client 6 sending request: {:jsonrpc "2.0", :method :workspace/executeCommand, :params {:command "drag-forward", :arguments ("file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/code_action/a.clj" 5 4)}, :id 3}                                                                                                                                                                                                                                                                                                   |
|   7 |  5.410 | 2022-09-16T06:50:49.6418436Z 2022-09-16T06:50:49.641494 Client 7 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}, :capabilities {:textDocument {:completion {:contextSupport true, :completionItem {:snippetSupport true, :commitCharactersSupport true, :preselectSupport true, :documentationFormat ["markdown" "plaintext"], :resolveSupport {:properties ["documentation" "detail" "additionalTextEdits"]}}}}}}, :id 1} |
|   7 |  0.120 | 2022-09-16T06:50:55.0538526Z 2022-09-16T06:50:55.052316 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 2, :character 3}}, :id 2}                                                                                                                                                                                                                                                                                       |
|   7 |  0.023 | 2022-09-16T06:50:55.1808227Z 2022-09-16T06:50:55.173229 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 2, :character 4}}, :id 3}                                                                                                                                                                                                                                                                                       |
|   7 |  0.005 | 2022-09-16T06:50:55.1983854Z 2022-09-16T06:50:55.196963 Client 7 sending request: {:jsonrpc "2.0", :method :textDocument/completion, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/completion/a.clj"}, :position {:line 4, :character 3}}, :id 4}                                                                                                                                                                                                                                                                                       |
|   8 |  5.210 | 2022-09-16T06:50:55.2217439Z 2022-09-16T06:50:55.220509 Client 8 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|   9 |  5.086 | 2022-09-16T06:51:00.9778097Z 2022-09-16T06:51:00.976427 Client 9 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  10 |  5.202 | 2022-09-16T06:51:06.6323401Z 2022-09-16T06:51:06.63211 Client 10 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :linters {:clj-kondo {:report-duplicates false}}}}, :id 1}                                                                                                                                                                                                                                                 |
|  11 |  5.258 | 2022-09-16T06:51:12.3825201Z 2022-09-16T06:51:12.382301 Client 11 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  12 |  5.355 | 2022-09-16T06:51:18.1603201Z 2022-09-16T06:51:18.157564 Client 12 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :clj-depend {:layers {:a {:defined-by ".*\\.depend\\.a", :accessed-by-layers #{}}, :b {:defined-by ".*\\.depend\\.b", :accessed-by-layers #{:c}}}}}}, :id 1}                                                                                                                                              |
|  13 |  5.158 | 2022-09-16T06:51:24.0671782Z 2022-09-16T06:51:24.065775 Client 13 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  14 |  5.246 | 2022-09-16T06:51:31.3789913Z 2022-09-16T06:51:31.37876 Client 14 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  14 |  0.097 | 2022-09-16T06:51:36.6281944Z 2022-09-16T06:51:36.625433 Client 14 sending request: {:jsonrpc "2.0", :method :textDocument/formatting, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/formatting.clj"}, :options {:tabSize 2, :insertSpaces true}}, :id 2}                                                                                                                                                                                                                                                                                |
|  14 |  0.030 | 2022-09-16T06:51:36.7316275Z 2022-09-16T06:51:36.731268 Client 14 sending request: {:jsonrpc "2.0", :method :textDocument/rangeFormatting, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/formatting.clj"}, :options {:tabSize 2, :insertSpaces true}, :range {:start {:line 6, :character 4}, :end {:line 7, :character 11}}}, :id 3}                                                                                                                                                                                                   |
|  15 |  5.305 | 2022-09-16T06:51:36.7954335Z 2022-09-16T06:51:36.794736 Client 15 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  15 |  0.131 | 2022-09-16T06:51:42.1053513Z 2022-09-16T06:51:42.103875 Client 15 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 15, :character 0}, :newName "::click-now"}, :id 2}                                                                                                                                                                                                                                                                    |
|  16 |  5.258 | 2022-09-16T06:51:42.2613842Z 2022-09-16T06:51:42.26096 Client 16 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  16 |  0.193 | 2022-09-16T06:51:47.5335877Z 2022-09-16T06:51:47.521787 Client 16 sending request: {:jsonrpc "2.0", :method :textDocument/prepareRename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 6}}, :id 2}                                                                                                                                                                                                                                                                                     |
|  16 |  0.010 | 2022-09-16T06:51:47.7149142Z 2022-09-16T06:51:47.714728 Client 16 sending request: {:jsonrpc "2.0", :method :textDocument/prepareRename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 15}}, :id 3}                                                                                                                                                                                                                                                                                    |
|  17 |  5.211 | 2022-09-16T06:51:47.7514594Z 2022-09-16T06:51:47.750022 Client 17 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  17 |  0.218 | 2022-09-16T06:51:52.9673017Z 2022-09-16T06:51:52.966264 Client 17 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 1, :character 36}, :newName "spec"}, :id 2}                                                                                                                                                                                                                                                                           |
|  18 |  5.239 | 2022-09-16T06:51:53.2158883Z 2022-09-16T06:51:53.214489 Client 18 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  18 |  0.166 | 2022-09-16T06:51:58.4767191Z 2022-09-16T06:51:58.458828 Client 18 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 12, :character 15}, :newName ":your-key"}, :id 2}                                                                                                                                                                                                                                                                     |
|  19 |  5.402 | 2022-09-16T06:51:58.6500736Z 2022-09-16T06:51:58.649703 Client 19 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  19 |  0.174 | 2022-09-16T06:52:04.0655865Z 2022-09-16T06:52:04.06058 Client 19 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/b.cljc"}, :position {:line 3, :character 4}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                        |
|  20 |  5.131 | 2022-09-16T06:52:04.2502763Z 2022-09-16T06:52:04.24997 Client 20 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  20 |  0.172 | 2022-09-16T06:52:09.3883368Z 2022-09-16T06:52:09.38489 Client 20 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/a.cljc"}, :position {:line 3, :character 9}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                        |
|  21 |  5.342 | 2022-09-16T06:52:09.5875084Z 2022-09-16T06:52:09.586371 Client 21 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  21 |  0.198 | 2022-09-16T06:52:14.9375232Z 2022-09-16T06:52:14.932902 Client 21 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/single_a.clj"}, :position {:line 2, :character 0}, :newName "::click-now"}, :id 2}                                                                                                                                                                                                                                                               |
|  22 |  5.338 | 2022-09-16T06:52:15.1567245Z 2022-09-16T06:52:15.15542 Client 22 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  22 |  0.189 | 2022-09-16T06:52:20.5007915Z 2022-09-16T06:52:20.497078 Client 22 sending request: {:jsonrpc "2.0", :method :textDocument/rename, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/rename/b.cljc"}, :position {:line 1, :character 52}, :newName "your-func"}, :id 2}                                                                                                                                                                                                                                                                      |
|  23 |  5.212 | 2022-09-16T06:52:20.7123375Z 2022-09-16T06:52:20.710989 Client 23 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  23 |  0.056 | 2022-09-16T06:52:25.9271892Z 2022-09-16T06:52:25.923821 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 7, :character 2}}, :id 2}                                                                                                                                                                                                                                                                       |
|  23 |  0.013 | 2022-09-16T06:52:26.0006979Z 2022-09-16T06:52:25.990388 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 2, :character 25}}, :id 3}                                                                                                                                                                                                                                                                      |
|  23 |  0.005 | 2022-09-16T06:52:26.0215787Z 2022-09-16T06:52:26.020178 Client 23 sending request: {:jsonrpc "2.0", :method :textDocument/documentHighlight, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_highlight/a.clj"}, :position {:line 14, :character 4}}, :id 4}                                                                                                                                                                                                                                                                      |
|  24 |  5.366 | 2022-09-16T06:52:26.0550610Z 2022-09-16T06:52:26.054859 Client 24 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  24 |  0.095 | 2022-09-16T06:52:31.4259190Z 2022-09-16T06:52:31.423274 Client 24 sending request: {:jsonrpc "2.0", :method :textDocument/documentSymbol, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/document_symbol/a.clj"}}, :id 2}                                                                                                                                                                                                                                                                                                                |
|  25 |  5.342 | 2022-09-16T06:52:31.6453984Z 2022-09-16T06:52:31.644153 Client 25 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  25 |  0.132 | 2022-09-16T06:52:36.9900100Z 2022-09-16T06:52:36.989686 Client 25 sending request: {:jsonrpc "2.0", :method :textDocument/linkedEditingRange, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/linked_editing_range/a.cljc"}, :position {:line 2, :character 13}}, :id 2}                                                                                                                                                                                                                                                                  |
|  25 |  0.006 | 2022-09-16T06:52:37.1223377Z 2022-09-16T06:52:37.121918 Client 25 sending request: {:jsonrpc "2.0", :method :textDocument/linkedEditingRange, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/linked_editing_range/a.cljc"}, :position {:line 2, :character 7}}, :id 3}                                                                                                                                                                                                                                                                   |
|  26 |  5.274 | 2022-09-16T06:52:37.1539620Z 2022-09-16T06:52:37.15358 Client 26 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                   |
|  26 |  0.050 | 2022-09-16T06:52:42.4320609Z 2022-09-16T06:52:42.428927 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 1, :character 2}}, :id 2}                                                                                                                                                                                                                                                                                     |
|  26 |  0.008 | 2022-09-16T06:52:42.4804744Z 2022-09-16T06:52:42.479295 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 3, :character 2}}, :id 3}                                                                                                                                                                                                                                                                                     |
|  26 |  0.007 | 2022-09-16T06:52:42.4890456Z 2022-09-16T06:52:42.487278 Client 26 sending request: {:jsonrpc "2.0", :method "clojure/cursorInfo/raw", :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/cursor_info/a.clj"}, :position {:line 2, :character 8}}, :id 4}                                                                                                                                                                                                                                                                                     |
|  27 |  5.281 | 2022-09-16T06:52:42.5199457Z 2022-09-16T06:52:42.518748 Client 27 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false}}, :id 1}                                                                                                                                                                                                                                                                                                               |
|  27 |  0.059 | 2022-09-16T06:52:47.8054927Z 2022-09-16T06:52:47.800851 Client 27 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 7, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  28 |  5.151 | 2022-09-16T06:52:47.8858826Z 2022-09-16T06:52:47.884325 Client 28 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :dependency-scheme "jar", :java {:decompile-jar-as-project? false}}}, :id 1}                                                                                                                                                                                                                                           |
|  28 |  1.531 | 2022-09-16T06:52:53.0665434Z 2022-09-16T06:52:53.036007 Client 28 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 8, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  28 |  0.005 | 2022-09-16T06:52:54.5685184Z 2022-09-16T06:52:54.567759 Client 28 sending request: {:jsonrpc "2.0", :method "clojure/dependencyContents", :params {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/.lsp/.cache/java/decompiled/clojure/lang/PersistentVector.java"}, :id 3}                                                                                                                                                                                                                                                                                                       |
|  29 |  5.331 | 2022-09-16T06:52:54.6148347Z 2022-09-16T06:52:54.613218 Client 29 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :dependency-scheme "zipfile", :java {:decompile-jar-as-project? false}}}, :id 1}                                                                                                                                                                                                                                       |
|  29 |  1.520 | 2022-09-16T06:52:59.9481416Z 2022-09-16T06:52:59.944939 Client 29 sending request: {:jsonrpc "2.0", :method :textDocument/definition, :params {:textDocument {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/src/sample_test/java_interop/a.clj"}, :position {:line 8, :character 5}}, :id 2}                                                                                                                                                                                                                                                                                    |
|  29 |  0.011 | 2022-09-16T06:53:01.4763047Z 2022-09-16T06:53:01.471505 Client 29 sending request: {:jsonrpc "2.0", :method "clojure/dependencyContents", :params {:uri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/.lsp/.cache/java/decompiled/clojure/lang/PersistentVector.java"}, :id 3}                                                                                                                                                                                                                                                                                                       |
|  30 |  7.973 | 2022-09-16T06:53:01.5373662Z 2022-09-16T06:53:01.537123 Client 30 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false, :stubs {:generation {:namespaces #{"datomic.api"}}}}}, :id 1}                                                                                                                                                                                                                                             |
|  31 |  9.051 | 2022-09-16T06:53:15.0446511Z 2022-09-16T06:53:15.044424 Client 31 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                  |
|  31 |  0.005 | 2022-09-16T06:53:24.0971883Z 2022-09-16T06:53:24.09582 Client 31 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
|  32 | 22.784 | 2022-09-16T06:53:24.1230942Z 2022-09-16T06:53:24.122679 Client 32 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-boot/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                             |
|  32 |  0.018 | 2022-09-16T06:53:46.9087523Z 2022-09-16T06:53:46.90738 Client 32 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
|  33 | 24.797 | 2022-09-16T06:53:46.9533659Z 2022-09-16T06:53:46.952948 Client 33 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-shadow/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                           |
|  33 |  0.008 | 2022-09-16T06:54:11.7517655Z 2022-09-16T06:54:11.750267 Client 33 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
|  34 |  9.407 | 2022-09-16T06:54:11.7840359Z 2022-09-16T06:54:11.782647 Client 34 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-lein/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                             |
|  34 |  0.012 | 2022-09-16T06:54:21.1910150Z 2022-09-16T06:54:21.189946 Client 34 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
|  35 | 16.253 | 2022-09-16T06:54:21.2409725Z 2022-09-16T06:54:21.23957 Client 35 sending request: {:jsonrpc "2.0", :method :initialize, :params {:rootUri "file:///home/runner/work/clojure-lsp/clojure-lsp/cli/integration-test/sample-test-bb/", :initializationOptions {:lint-project-files-after-startup? false, :java false}}, :id 1}                                                                                                                                                                                                                                                                                                |
|  35 |  0.007 | 2022-09-16T06:54:37.4936159Z 2022-09-16T06:54:37.492643 Client 35 sending request: {:jsonrpc "2.0", :method "clojure/serverInfo/raw", :params {}, :id 2}                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Reply:
Got it, so since it's a windows issue, could we add a if there to make sure the timeout is greater only on windows?

Reply:
> Got it, so since it's a windows issue, could we add a if there to make sure the timeout is greater only on windows? 

Done. Please note the slowness is also present in macos builds.